presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -514,6 +513,48 @@ private static Slice formatDatetime(ISOChronology chronology, Locale locale, lon
         }
     }
 
+    @Description("Returns the current Unix timestamp in seconds (seconds since Unix epoch)")
+    @ScalarFunction
+    @SqlType(StandardTypes.BIGINT)
+    public static long unixTimestamp(ConnectorSession session)
+    {
+        return MILLISECONDS.toSeconds(session.getStartTime());
```

Comment:
Are you sure, that you should use `getStartTime()` here? Isn't `startTime` fixed for the duration of the session, so following sql's run in presto-cli will return the same result?:
```
select unix_timestamp(); -- we get some result
-- we wait 10 seconds
select unix_timestamp(); -- despite 10 seconds delay, we get same timestamp
```